src/ethereum/utils/address.py

```diff
@@ -0,0 +1,34 @@
+"""
+Utility Functions For Addresses
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+..contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+Address specific utility functions used in this application.
+"""
+from typing import Union
+
+from ethereum.base_types import U256, Uint
+from ethereum.frontier.eth_types import Address
```

Comment:
Something in the "parent" module probably shouldn't import something from inside the sub-module.

Case in point, when we introduce  the homestead fork, mypy will complain if we use homestead's `Address` type in `to_address`.

tl;dr `to_address` should be in frontier.